compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -1423,9 +1423,9 @@ object Parsers {
         case FORSOME => syntaxError(ExistentialTypesNoLongerSupported()); t
         case _ =>
           if (imods.isOneOf(GivenOrImplicit) && !t.isInstanceOf[FunctionWithMods])
-            syntaxError("Types with implicit keyword can only be function types `implicit (...) => ...`", implicitKwPos(start))
+            syntaxError(ImplicitTypesCanOnlyBeFunctionTypes())
```

Comment:
It's nicer when the error message highlights the keyword in question rather than just points to the location after the erroneous type.

```suggestion
            syntaxError(ImplicitTypesCanOnlyBeFunctionTypes(), implicitKwPos(start))
```